etc/make_release.py

```diff
@@ -224,6 +227,65 @@ def release(jira_creds_file,
                                     release_notes_text, output_file, quiet)
 
 
+def check_docker_file_versions(mongo_cxx_release_ver: str):
+    """
+    Checks that `MONGOC_VERSION` defined in Dockerfiles matches `LIBMONGOC_REQUIRED_VERSION` from CMakeLists.txt.
+    Checks that `MONGOCXX_VERSION` defined in Docker Makefiles matches version to be released: `mongo_cxx_release_ver`.
+    """
+
+    dockerfiles = [
+        "./extras/docker/alpine3.18/Dockerfile",
+        "./extras/docker/bookworm/Dockerfile",
+        "./extras/docker/jammy/Dockerfile",
+        "./extras/docker/redhat-ubi-9.2/Dockerfile",
+    ]
+    makefiles = [
+        "./extras/docker/alpine3.18/Makefile",
+        "./extras/docker/bookworm/Makefile",
+        "./extras/docker/jammy/Makefile",
+        "./extras/docker/redhat-ubi-9.2/Makefile",
+    ]
```

Comment:
I wonder if something like this might be better:

```
extras_docker = pathlib.Path("./extras/docker")
dockerfiles = extras_docker.rglob("Dockerfile")
makefiles = extras_docker.rglob("Makefile")
```

That should make additions of other distros work automagically, and it should allow the code that follows below (i.e., the `for` loops) to work without further modification. It's also more concise.

Reply:
I like it, I pushed this change.